ci/ray_ci/BUILD.bazel

```diff
@@ -29,15 +29,17 @@ py_library(
 )
 
 py_binary(
-    name = "ray_ci",
-    srcs = ["ray_ci.py"],
-    deps = [":ray_ci_lib"],
+    name = "test_in_docker",
+    srcs = ["test_in_docker.py"],
+    deps = [
+        ":ray_ci_lib",
+    ],
 )
 
 py_test(
-    name = "test_runner",
+    name = "test_test_runner",
     size = "small",
-    srcs = ["test_runner.py"],
+    srcs = ["tests/test_test_runner.py"],
```

Comment:
lol.. how many `test` can you have in a line?

how about this? instead of calling this `test_runner.py`, call it something like `tester.py` and/or `docker_tester.py`, `container_tester.py`, and then you can have unit tests called `test_docker_tester.py`, which feels a bit better?

conventionally, bazel unit tests lives in the same directory as the source files. the colocation makes the tests easier to find.